### PR TITLE
enh(form) : removed centreon-cloud form on configuration > users > contact/user (MON-12218)

### PR DESCRIPTION
**Pull request by AssanDiomande**
## Description

**Fixes** MON-12445

Removed centreon-cloud contact form on centreon-web
More informations about the new form [here](https://centreon.atlassian.net/browse/MON-12218)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

Go to the how to test section in the associated [jira ticket](https://centreon.atlassian.net/browse/MON-12445)

### CODE REVIEW

File: www/include/configuration/configObject/contact/formContact.php [hunk 1]
```diff
@@ -505,12 +484,12 @@
...
-        'contact_admin' => ['contact_admin' => '0'],
-        'reach_api' => ['reach_api' => '0'],
-        'reach_api_rt' => ['reach_api_rt' => '0']
-    ]);
+    $form->setDefaults(array(
+        'contact_oreon' => '1',
+        'contact_admin' => '0',
+        'reach_api' => '0',
+        'reach_api_rt' => '0'
+    ));
```
Comment (kduret) [inline www/include/configuration/configObject/contact/formContact.php:492]: you should keep this king of bugfix ;)

Comment (AssanDiomande) [reply www/include/configuration/configObject/contact/formContact.php]: done

**Review by jeremyjaouen — DISMISSED** [review]

File: www/include/configuration/configObject/contact/formContact.php [hunk 2]
```diff
@@ -75,24 +75,14 @@
...
-    if ($result['key'] === 'isRemote' && $result['value'] === 'yes') {
-        $platformType = PLATFORM_REMOTE;
-    }
+$dbResult = $pearDB->query("SELECT i.value FROM informations i WHERE i.key = 'isRemote'");
+$result = $dbResult->fetch();
+if ($result === false) {
+    $isRemote = false;
+} else {
+    $isRemote = array_map("myDecode", $result);
+    $isRemote = ($isRemote['value'] === 'yes') ? true : false;
```
Comment (kduret) [inline www/include/configuration/configObject/contact/formContact.php:84]:
```suggestion
    $isRemote = $isRemote['value'] === 'yes';
```

Comment (AssanDiomande) [reply www/include/configuration/configObject/contact/formContact.php]: done

**Review by kduret — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**